src/collective/nitf/tests/test_content.robot

```diff
@@ -23,9 +23,9 @@ Test Folder Full View
     Open Add New Menu
     Click Link  css=a#folder
     Page Should Contain  Add Folder
-    Input Text  css=#form-widgets-IDublinCore-title  Test Folder
+    Input Text  css=#form-widgets-IDublinCore-title,#title  Test Folder
```

Comment:
maybe we can make some tests with last version of https://pypi.python.org/pypi/cssselect package to see if all css selectos work with RF, we are using the version *0.3* instead of *1.0.0*